src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--month-length-30">
+      <div class="calendar__day calendar__day--1 calendar__day--start-day-sun"></div>
+      <div class="calendar__day calendar__day--2"></div>
```

Comment:
start-day should be a modifier for the calendar
don't need to add 1, 2, 3 modifiers for days. You can use nth-child

```suggestion
    <section class="calendar calendar--month-length-30 calendar--start-day-sun">
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
Зрозуміла, дякую) Все змінила.